src/App/DsiplayUsers/DisplayUsers.tsx

```diff
@@ -0,0 +1,106 @@
+import { Container, List, ListItem, makeStyles, Theme, Typography } from '@material-ui/core'
+import { useContext, useEffect, useState } from 'react'
+import { GlobalContext } from '../../GlobalContext'
+import { theme } from '../../theme'
+import { GetUsersEndpointResult } from '../../services/ApiService'
+
+function DisplayUsers() {
```

Comment:
Since this is a page, let's keep the `Page` name convention. Also, our components are generally stored into a const, which then gets exported at the bottom of the file

```suggestion
const DisplayUsersPage: FC = () => {
```

Reply:
Updated in all locations